README.md

```diff
@@ -263,14 +263,34 @@ The following flags are available for `organizations get`:
 
 * Get an organization by its externalID: `ocm support get organizations [organizationExternalID]`
 * Get an organization and include its subscriptions: `ocm support get organizations [organizationID] --fetchSubscriptions`
-* Get an organization and include its labels `ocm support get organizations [organizationID] --fetchLabels --fetchCapabilities`
+* Get an organization and include its labels `ocm support get organizations [organizationID] --fetchLabels`
 * Get an organization and include its capabilities `ocm support get organizations [organizationID] --fetchCapabilities`
 * Get an organization and include its quota: `ocm support get organizations [organizationID] --fetchQuota`
-<<<<<<< HEAD
-* Get an organization and include its resource quota: `ocm support get organizations [organizationID] --fetchQuota`
-=======
 * Get an organization and include its resource quota: `ocm support get organizations [organizationID] --fetchSkus`
->>>>>>> ad8174efc59af98400dd72311c9931dea4e700b1
+
+#### Getting a subscription
+
+Use the `subscriptions` subcommand to get one or more subscriptions, passing as argument one of the following:
+
+* subscriptionID
+* cluster_id
+* external_cluster_id
+* organization_id
+
+The following flags are available for `subscriptions get`:
+
+```
+--all                  If true, returns all organizations that matched the search instead of the first one only (default behaviour).
+--fetchLabels          If true, includes the organization labels.
+--fetchCapabilities    If true, includes the organization capabilities.
+-h, --help             help for get
+```
```

Comment:
in addition to fetchLabels and fetchCapabilities, let's add fetchAccounts fetchCpuAndSocket - please see API docs for reference.